client/assets/sass/_overrides.sass

```diff
@@ -41,12 +42,25 @@
   .toolbar-pf-actions
 
     .form-group
-      margin-bottom: 0 !important // sass-lint:disable-line no-important
+      margin-bottom: 0 !important
+
+    // Override margin-left added to parent of pfFilter complex select
+    .btn-group + .dropdown
+      margin-left: 0
+
+    // pf-toolbar-dropdown goes crazy when given sufficent cause to, this keeps it inline
+    .toolbar-apf-filter
+      .dropdown-menu
+        max-height: 78em
+        max-width: 202px
+        overflow-y: scroll
+
+        li
+          overflow-x: scroll
 
 // Override patternfly-timeline default icon size
 .timeline-pf-drop
-  font-size: 20px !important // sass-lint:disable-line no-important
+  font-size: 20px !important
```

Comment:
Again, already in place, just reformatted,  only use of !important is in the overrides file, what was added is that mess on line 47, tried to override the styles being applied but wasn't able to, cleanest fix was to replace them exactly with a non-value, checked around the rest of the app, doesn't appear to have any ill affects (*yet*)